### ISSUE TITLE
No such file or directory: '.../french-stemmer.js'

### ISSUE THREAD

**Issue by ymoisan** [sphinx-doc/sphinx#2362]
Hi All,

I tried to build a project I haven't built in a while with Sphinx 1.3.1+.  It used to be built on another machine with Sphinx 1.2.x.  I get that error : 

```
copying static files... 
Exception occurred:
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: '/usr/local/lib/python2.7/dist-packages/sphinx/search/non-minified-js/french-stemmer.js'
```

Ubuntu 14.04.  Full trace : 
[sphinx-err-ugLxfI.log.txt](https://github.com/sphinx-doc/sphinx/files/152982/sphinx-err-ugLxfI.log.txt)

I don't get the problem when I set

```
language = None

...
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.
```

instead of 

```
language = 'fr'
```

in conf.py and I still get the search box to find accented letters even though the build says "dumping search index in English".

There is something missing with the French language pack.  How can I check that or reinstall French support ?

I re-ran sphinx-quickstart on my new machine and pasted the contents for my old project and I still get the same error unless I comment out the language directive in conf.py.

Thanx

**Comment by ymoisan** [comment]
```
pip list --outdated
```

...
Sphinx (Current: 1.2.2 Latest: 1.3.6)

Re-installed with

```
sudo pip install -U Sphinx
```

and now everything is OK.

```
pip show sphinx
```

still mentions 1.2.2 but pip did download 1.3.6 and I can see in my generated docs.  Weird, but it works.

**Comment by shimizukawa** [comment]
The issue was appeared since 1.3 release and it has been fixed at 1.3.3.
https://github.com/sphinx-doc/sphinx/commit/f09ca2c18693686a1f3ab8096eaec16b4fd7213f

Anyhow, thanks for you reporting!
